src/win/util.c

```diff
@@ -1171,18 +1171,18 @@ int uv_os_homedir(char* buffer, size_t* size) {
 
 
 int uv_os_tmpdir(char* buffer, size_t* size) {
-  wchar_t path[MAX_PATH + 1];
+  wchar_t path[MAX_PATH + 2];
   DWORD bufsize;
   size_t len;
 
   if (buffer == NULL || size == NULL || *size == 0)
     return UV_EINVAL;
 
-  len = GetTempPathW(MAX_PATH + 1, path);
+  len = GetTempPathW(MAX_PATH + 2, path);
```

Comment:
Unrelated to the issue at hand, but maybe this and line 1185 could use `sizeof`.

Reply:
It should be `ARRAY_SIZE(path)` because it's an array of `wchar_t` but otherwise I agree: DRY.